.eslintrc.js

```diff
@@ -84,6 +84,15 @@ const INLINE_NON_VOID_ELEMENTS = [
         'no-undef': 'off',
         // Turns off some non-TypeScript rules in favor of their specific TypeScript rules to avoid false negatives:
         indent: 'off',
+        '@typescript-eslint/type-annotation-spacing': ['error', {
+          before: false,
+          after: true,
+          overrides: {
+            arrow: {
+              before: true,
+            },
+          },
+        }],
```

Comment:
**Suggestion**: Please move this to another place that’s not in between the two indent rules. Those logically relate to each other so it’s nice to keep them next to each other, too.

Reply:
Oh! Sure yeah, thats done!